src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance;
+
+    private HibernateUtil() {
+        instance = new Configuration().configure().buildSessionFactory();
+    }
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        if (instance == null) {
+            new HibernateUtil();
```

Comment:
Better have an empty constructor and use initFactory() method. 
Or use eager initialization.
Initializing **static** fields in the constructor is a bad idea.